contracts/mothership/OverlayV1Mothership.sol

```diff
@@ -60,65 +60,59 @@ contract OverlayV1Mothership is AccessControlEnumerable {
 
         _setupRole(ADMIN, msg.sender);
         _setupRole(GOVERNOR, msg.sender);
-        _setupRole(GUARDIAN, msg.sender);
```

Comment:
Why are we removing the guardian role? I don't have any strong opinions yet about the types of roles we need but it is likely more than just admin and governor.

Governor would be something with a reasonable time lock for instance. 

Guardian would be something along the lines of emergency shutdown. 

Admin, that'd be to grant/remove various roles. 

Not sure though. I'd like to look at other protocol's thought processes. Fei in particular. We are in the realm of somewhat more permissioned operation so maybe protocols like Aave and Compound should also be taken into account.